components/CreativeCommonsNotice.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const creativeCommonsUrl = 'https://creativecommons.org/licenses/by-nc-sa/4.0/'
+
+const CreativeCommonsLicense = () => (
+    <div className="my-3">
+        <div className="mb-3 text-muted small">
+            This work is licensed under a{' '}
+            <a href={creativeCommonsUrl} className="text-muted">
```

Comment:
Could we make this link the same style as the rest of the footer?

```
className="text-mutedext-dark text-decoration-none"
```

Reply:
I implemented this suggestion. I noticed that `text-muted` overrides the effect of `text-dark` so I omitted the latter class.